common/app/model/trails.scala

```diff
@@ -45,9 +47,16 @@ trait Trail extends Elements with Tags with FaciaFields with Dates {
     case snap: Snap => snap.snapHref.filter(_.nonEmpty)
     case t: Trail => Option(t.url)
   }
+
+  def trailSlideshow(aspectWidth: Int, aspectHeight: Int): Iterable[FaciaImageElement] =
+  slideshow
+    .filter(image => IsRatio(5, 3, image.width, image.height))
```

Comment:
Maybe I'm missing something, but should 5, 3 actually be aspectWidth, aspectHeight?